lib/faker/default/date.rb

```diff
@@ -37,9 +39,11 @@ def between(legacy_from = NOT_GIVEN, legacy_to = NOT_GIVEN, from:, to:)
       # @param excepted [Date, String] A date to exclude.
       # @return [Date]
       #
-      # @example
-      #   Faker::Date.between_except(from: 1.year.ago, to: 1.year.from_now, excepted: Date.today)
-      #     #=> #<Date: 2014-10-03>
+      # @example if used with or without Rails (Activesupport)
```

Comment:
> The proper names of Rails components have a space in between the words, like “Active Support”. ActiveRecord is a Ruby module, whereas Active Record is an ORM.

https://github.com/rails/rails/blob/6bb7d50dece5d02540e5c420f63478185102fd70/guides/source/api_documentation_guidelines.md#wording

```suggestion
      # @example if used with or without Rails (Active Support)
```

Reply:
Would you like to make similar changes to entire this PR?

Reply:
Sure, thanks for the suggestion!